notebooks/official/bigquery_ml/get_started_with_bqml_training.ipynb

```diff
@@ -320,7 +320,7 @@
       },
       "outputs": [],
       "source": [
-        "! gsutil mb -l $REGION $BUCKET_URI"
+        "! gsutil mb -l {REGION} $BUCKET_URI"
```

Comment:
question: I'm not sure that this will work. Won't the base interpreter evaluate this as a string rather than the variable `REGION`?